src/App.js

```diff
@@ -1,5 +1,90 @@
+import { MissionUtils } from "@woowacourse/mission-utils";
+
 class App {
-  async play() {}
+
+  getRandomNumber(){
+    const computerNumber = [];
+    while (computerNumber.length < 3){
+      const number = MissionUtils.Random.pickNumberInRange(1, 9);
+      if(!computerNumber.includes(number))
+        computerNumber.push(number);
+    }
+    return computerNumber;
+  }
+
+  async getUserInput(){
+    const userInput = await MissionUtils.Console.readLineAsync("숫자를 입력해주세요 : ");
+    const validatedUserInput = this.checkInputValidation(userInput);
+
+    return validatedUserInput;
+  }
+
+  checkInputValidation(userInputNumber){
+    const userInputNumberArr = userInputNumber.split('').map(Number);
+
+    if(userInputNumber.length !== 3) throw new Error("[ERROR] 3자리 숫자가 아닙니다.");
+    if(new Set(userInputNumberArr).size !== 3) throw new Error("[ERROR] 중복된 숫자가 있습니다.");
+    if(userInputNumber.includes(0)) throw new Error("[ERROR] 1~9 사이의 숫자가 아닙니다.");
+    if(isNaN(userInputNumber)) throw new Error("[ERROR] 숫자가 아닙니다.");
```

Comment:
전역 isNaN은 숫자가 아닌 것을 숫자로 강제하고, NaN으로 간주되는 모든 것을 true로 반환해서 의도치 않은 결과가 나올 수 있다고 해요!

```suggestion
    if(Number.isNaN(userInputNumber)) throw new Error("[ERROR] 숫자가 아닙니다.");
```

[참고 자료]
https://github.com/parksb/javascript-style-guide#standard-library--isnan